examples/shopping-cart/src/actions/index.js

```diff
@@ -17,12 +17,23 @@ const addToCartUnsafe = productId => ({
   productId
 })
 
+const removeFromCartUnsafe = (productId,quantityById) => ({
```

Comment:
Hi Tim, Thanks for reviewing my code. I was following the naming convention of the **addToCartUnsafe**, but happy to rename the method, what is a better name for that method ? We already have **removeFromCart** so let me know, thanks... Also, the tests are failing, so I am working on fixing the tests as well.  I will do another commit once I fix the failing tests.  Stay tuned...